__all__ = ["CustomError"]


class CustomError(Exception):
    pass
